src/main/java/frc/trigon/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -101,8 +100,6 @@ private static void configureMasterMotor() {
         MASTER_MOTOR.registerSignal(TalonFXSignal.STATOR_CURRENT, 100);
         MASTER_MOTOR.registerSignal(TalonFXSignal.TORQUE_CURRENT, 100);
         MASTER_MOTOR.registerSignal(TalonFXSignal.MOTOR_VOLTAGE, 100);
-        MASTER_MOTOR.registerSignal(TalonFXSignal.REVERSE_LIMIT, 100);
```

Comment:
The climbers can't use the signal from the limit switch if it isn't registered. You still need the reverse limit.